src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -162,3 +162,23 @@ def validate_asp_sku(cmd, namespace):
         if res.get('properties').get('hostingEnvironment') is not None:
             raise CLIError("Only pricing tier 'Isolated' is allowed in this app service plan. Use this link to "
                            "learn more: https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans")
+
+
+def validate_ip_address(namespace):
+    if namespace.ip_address is not None:
+        # IPv6
+        if ':' in namespace.ip_address:
```

Comment:
just curious, is there existing python lib for ip address validation?

Reply:
Hi @yungezz. There is an ipaddress library (https://docs.python.org/3/library/ipaddress.html) and I had also evaluated it, but did not find it fit for the purpose. For a simple validation and ensuring that a single IP is converted to a CIDR, the code should handle it.